static/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx

```diff
@@ -221,15 +221,17 @@ class FiltersAndSampling extends AsyncView<Props, State> {
             }
           )}
         </TextBlock>
-        <RulesPanel
-          rules={errorRules}
-          disabled={disabled}
-          onAddRule={this.handleAddRule('errorRules')}
-          onEditRule={this.handleEditRule}
-          onDeleteRule={this.handleDeleteRule}
-          onUpdateRules={this.handleUpdateRules}
-          isErrorPanel
-        />
+        {organization.features.includes('filters-and-sampling-error-rules') && (
```

Comment:
We usually use `<Feature/>` component to feature gate something in the markup. 🤷‍♂️

Reply:
sure ... but in this case we don't want to display anything, just hide the feature